README.md

```diff
@@ -14,45 +14,58 @@ The toolkit library jar is hosted on https://esri.jfrog.io/artifactory/arcgis.
 
 To add the dependency to your project using Gradle:
 ```groovy
-apply plugin: 'application'
-
-// Runtime SDK dependency
-apply plugin: 'com.esri.arcgisruntime.java'
-buildscript {
-    repositories {
-        maven {
-            url 'https://esri.jfrog.io/artifactory/arcgis'
-        }
+plugins {
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.8'
+}
+
+// Replace with version number of ArcGIS SDK you are using in your app, such as:
+// arcgisVersion = '100.11.0'. See table below for SDK Versions that support the toolkit.
+ext {
+  arcgisVersion = '100.2.1'
+}
+
+javafx {
+    version = "11.0.2"
+    modules = [ 'javafx.controls' ]
+}
+
+compileJava.options.encoding = 'UTF-8'
+
+// Toolkit and Runtime SDK repository
+repositories {
+    jcenter()
+    maven {
+        url 'https://esri.jfrog.io/artifactory/arcgis'
     }
-    dependencies {
-        classpath 'com.esri.arcgisruntime:gradle-arcgis-java-plugin:1.0.0'
+    maven {
+        url 'https://olympus.esri.com/artifactory/arcgisruntime-repo'
     }
 }
```

Comment:
You don't need to include the olympus server.  Customers don't have access to this.  The only reason its included in the samples code is to allow us to develop samples against daily builds so our customers can see new functionality before it is released.